lib/document.d.ts

```diff
@@ -0,0 +1,46 @@
+import Container, { ContainerProps } from './container.js'
+import { ProcessOptions } from './postcss.js'
+import Result from './result.js'
+import Root, { RootProps } from './root.js'
+
+export interface DocumentProps extends ContainerProps {
+  nodes?: Root[]
+}
+
+type ChildNode = Root
+type ChildProps = RootProps
+
+/**
+ * Represents a file and contains all its parsed nodes.
+ *
+ * **Experimental!** Some aspects of this node could change within minor or patch version releases.
+ *
+ * ```js
+ * const document = postcss.parse('<html><style>a{color:black}</style><style>b{z-index:2}</style>')
```

Comment:
```suggestion
 * const document = htmlParser('<html><style>a{color:black}</style><style>b{z-index:2}</style>')
```

`postcss.parse` doesn’t have HTML support.